### PR TITLE
M4871 Amazon Bucket file ingest job for VKGL project

### PR DESCRIPTION
**Pull request by bartcharbon**
#### Checklist
- [ ] Functionality works & meets specifications
- [ ] Code reviewed
- [ ] Code unit/integration/system tested
- [ ] User documentation updated
- [ ] (If you have changed REST API interface) view-swagger.ftl updated
- [x] Test plan template updated ([Bucket_testplan.xlsx](https://github.com/molgenis/molgenis/files/1007033/Bucket_testplan.xlsx))
- [ ] Clean commits

### CODE REVIEW

**Comment by marikaris** [comment]
Testplan: " after the bucket in all three tests. In first test the target entity id should be removed

File: molgenis-amazon-bucket/src/main/java/org/molgenis/file/ingest/bucket/AmazonBucketConfig.java [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+
+	@Autowired
+	private AmazonBucketJobExecutionMetaData amazonBucketJobExecutionMetaData;
+
+	@Bean
+	public JobFactory amazonBucketJobFactory()
+	{
+		return new JobFactory()
+		{
+			@Override
```
Comment (fdlk) [inline molgenis-amazon-bucket/src/main/java/org/molgenis/file/ingest/bucket/AmazonBucketConfig.java]: we've separated JobFactory and ScheduledJobType into two separate beans.

File: molgenis-amazon-bucket/pom.xml [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+        <dependency>
+            <groupId>com.fasterxml.jackson.core</groupId>
+            <artifactId>jackson-core</artifactId>
+            <version>2.8.6</version>
+        </dependency>
+        <dependency>
+            <groupId>org.apache.poi</groupId>
+            <artifactId>poi</artifactId>
+            <version>${apache.poi.version}</version>
+        </dependency>
```
Comment (fdlk) [inline molgenis-amazon-bucket/pom.xml]: Are dependencies on poi and poi-ooxml still relevant?

**Comment by tommydeboer** [comment]
test this please

**Pull request merged**